### PR TITLE
Feature/add lisk

### PR DESCRIPTION
**Pull request by gost3228**

### CODE REVIEW

File: groups/lsk.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    SALT: 'adm',
+    ITERATIONS: 2048,
+    KEYLEN: 32,
+    DIGEST: 'sha256'
+}
+
+/**
+ * Generates a LSK account from the passphrase specified.
+ * @param {string} passphrase ADAMANT account passphrase
+ * @returns {object} network info, keyPair, privateKey, privateKeyWIF
```
Comment (adamant-al) [inline groups/lsk.js:18]: Update returned info

**Review by martiliones — CHANGES_REQUESTED** [review]

File: groups/lsk.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+const cryptography = require('@liskhq/lisk-cryptography')
+const sodium = require('sodium-browserify-tweetnacl')
+const pbkdf2 = require('pbkdf2');
+
+const coinNetworks = require('./coinNetworks');
+const { bytesToHex } = require("../helpers/encryptor");
```
Comment (martiliones) [inline groups/lsk.js:6]: use single quotes

**Review by martiliones — CHANGES_REQUESTED** [review]

**Comment by adamant-al** [comment]
@martiliones 
After this merge, make a branch with linter and tests

**Comment by martiliones** [comment]
@adamant-al I think we can just merge. In any case, the code will be refactored after setting the code style and linter for the whole project.

**Review by martiliones — APPROVED** [review]

**Review by MerdyNumber1 — APPROVED** [review]

File: groups/lsk.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+ */
+
+lsk.keys = passphrase => {
+  const network = coinNetworks.LSK;
+  const liskSeed = pbkdf2.pbkdf2Sync(passphrase, LiskHashSettings.SALT, LiskHashSettings.ITERATIONS, LiskHashSettings.KEYLEN, LiskHashSettings.DIGEST);
+  const keyPair = sodium.crypto_sign_seed_keypair(liskSeed);
+  const address = cryptography.getBase32AddressFromPublicKey(keyPair.publicKey);
+  const addressHexBinary = cryptography.getAddressFromPublicKey(keyPair.publicKey);
+  const addressHex = bytesToHex(addressHexBinary);
+  const privateKey = keyPair.secretKey.toString('hex');
```
Comment (adamant-al) [inline groups/lsk.js:30]: Add exporting privateKeyBinary

Comment (adamant-al) [reply groups/lsk.js:30]: Or wait, is it included in keyPair?

Comment (martiliones) [reply groups/lsk.js:30]: @adamant-al It is

**Pull request merged**